src/osemgrep/networking/Semgrep_App.mli

```diff
@@ -70,3 +73,8 @@ val fetch_scan_config_async :
   (Semgrep_output_v1_t.scan_config, string) result Lwt.t
 (** [fetch_scan_config_async ~token ~sca ~dry_run ~full_scan repo] returns a
      promise of the rules for the provided configuration. *)
+
```

Comment:
Where is it downloaded? Maybe called it fetch_pro_binary and add a comment
saying the content of the binary is in the body of the rsponse (in get_info).